packages/services/api/src/modules/audit-logs/resolvers/AppDeploymentPublishedAuditLog.ts

```diff
@@ -0,0 +1,24 @@
+import { AuditLogManager } from '../providers/audit-logs-manager';
+import type { AppDeploymentPublishedAuditLogResolvers } from './../../../__generated__/types.next';
+
+/*
+ * Note: This object type is generated because "AppDeploymentPublishedAuditLogMapper" is declared. This is to ensure runtime safety.
+ *
+ * When a mapper is used, it is possible to hit runtime errors in some scenarios:
+ * - given a field name, the schema type's field type does not match mapper's field type
+ * - or a schema type's field does not exist in the mapper's fields
+ *
+ * If you want to skip this file generation, remove the mapper or update the pattern in the `resolverGeneration.object` config.
+ */
+export const AppDeploymentPublishedAuditLog: AppDeploymentPublishedAuditLogResolvers = {
+  __isTypeOf: e => e.event_action === 'APP_DEPLOYMENT_PUBLISHED',
+  eventTime: e => new Date(e.event_time).toISOString(),
+  deploymentId: e => e.metadata.appDeploymentCreatedAuditLogSchema.deploymentId,
+  deploymentVersion: e => e.metadata.appDeploymentCreatedAuditLogSchema.deploymentVersion,
+  id: async (_parent, _arg, _ctx) => {
```

Comment:
should we do something about these generated resolvers? 
I think it needs to be implemented?